### PR TITLE
Added documentation for linuxlogo command 

### PR DESCRIPTION
**Pull request by WenhaoSu**
# Advanced HW 5 submission

Closes #[584]

Name: [Wenhao Su]
Uniqname: [suwenhao]

### CODE REVIEW

**Comment by cyanliu** [comment]
Before we can merge this, can you include documentation of the different flag options and maybe even include screenshots because some of the logos are rendered with color?

**Comment by WenhaoSu** [comment]
Screenshots and information for different flag options are added.

File: _commands/toys/linuxlogo.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+#### Example command
+
+~~~bash
+$ linuxlogo 
+~~~
+![linuxlogo](https://www.tecmint.com/wp-content/uploads/2015/06/Get-Default-OS-Logo.png)
+
+~~~bash
+$ linuxlogo -L 1
```
Comment (cyanliu) [inline _commands/toys/linuxlogo.md]: Adding a flag with just a screenshot isn't very helpful documentation, could you verbally describe what is happening as well? What does the -L flag add?

**Comment by cyanliu** [comment]
I've awarded you credit for advanced hw 5

**Pull request merged**